src/utility/instanceZUID.js

```diff
@@ -0,0 +1,2 @@
+const instanceZUID = window.location.host.split(".")[0];
```

Comment:
This happens on app boot and is added to the store. Can the store value be selected instead of adding this util?

Reply:
This was added in the previous RTK Query implementation and it is used to construct the RTK Query base url so I don't think I would be able to select it from the store